doc/overview/project_capabilities.md

```diff
@@ -12,6 +12,15 @@ known to declare the capability.
 | VisualC | Project may contain or compile C++ source files.
 | CSharp | Project may contain or compile C# source files.
 | VB | Project may contain or compile VB source files.
+| JavaScript | Project contains and handles JavaScript source files.
+| TypeScript | Indicates that the TypeScript build targets are imported into the project and the project is capable of compiling TypeScript source files into JavaScript.
+| JavaScriptProject | Indicates that the project supports JavaScript development. This capability also means that the project can manage JavaScript related project items and resolving JavaScript library references.
```

Comment:
Where are "javascript library references" supported? Wasn't this capability first used in Dev11? Were javascript library references supported back then?